.github/templates/main.yml

```diff
@@ -104,6 +115,11 @@ run: |
 shell: bash
 #@ end
 
+#@ def buildNetFrameworkTest():
```

Comment:
I deleted your comment by mistake, I meant to delete mine. Sorry about that. If you could re-add it, it'd be helpful for future refs, I suppose.
Anyway, my answer to that comment was the following:
Would you recommend sticking close to [what we do in jenkins](https://github.com/realm/realm-dotnet/blob/cfb988822d647ba3b6e2532cd1a01aaa03da8d39/Jenkinsfile#L411) or a more streamlined version of it?

Reply:
Yes something similar, but we can definitely simplify some things. For example, it can accept just a relative path to the project as we can always prefix it with `${{ github.workspace }}` - we'll never have to build outside of that. We can also always set the configuration. And the rest will have to be a map of properties that we just concat with `-p:*key*=*value*`.